resources/core/charts/docs/charts/content-ui/templates/docs-components-event-mesh-cag.yaml

```diff
@@ -5,13 +5,13 @@ metadata:
     rafter.kyma-project.io/view-context: docs-ui
     rafter.kyma-project.io/group-name: components
     rafter.kyma-project.io/order: "5"
-  name: event-mesh
+  name: eventing
```

Comment:
Does it make sense to change the file name from **docs-components-event-mesh-cag.yaml** to **docs-components-eventing-cag.yaml**.

Reply:
I changed the name in f76d100